x-pack/test/functional/apps/code/history.ts

```diff
@@ -200,22 +200,22 @@ export default function manageRepositoriesFunctionalTests({
         await PageObjects.common.sleep(5000);
         await testSubjects.click('codeStructureTreeTab');
         await retry.try(async () => {
-          const tabText = await (await find.byCssSelector('.euiTab-isSelected')).getVisibleText();
-          expect(tabText).to.equal('Structure');
+          // if structure tree tab is active, file tree tab's `data-test-subj` would be `codeFileTreeTab`
+          expect(testSubjects.exists('codeFileTreeTab')).to.be.ok();
```

Comment:
should we change to `codeStructureTreeTabActive` which is more readable?

Reply:
change to `codeStructureTreeTabActive` would affect `explore_repository` test, can we leave it for now

Reply:
ok